tests/testlib/s2n_test_certs.c

```diff
@@ -33,7 +33,7 @@ int s2n_test_cert_chain_and_key_new(struct s2n_cert_chain_and_key **chain_and_ke
     GUARD(s2n_read_test_pem(cert_chain_file, cert_chain_pem, S2N_MAX_TEST_PEM_SIZE));
     GUARD(s2n_read_test_pem(private_key_file, private_key_pem, S2N_MAX_TEST_PEM_SIZE));
 
-    notnull_check(*chain_and_key = s2n_cert_chain_and_key_new());
+    POSIX_GUARD_PTR(*chain_and_key = s2n_cert_chain_and_key_new());
```

Comment:
Why change just this notnull_check to POSIX_GUARD_PTR?

Reply:
`notnull_check` hides error messages and locations which makes it harder to debug issues. I had messed up one of the macros and using guard instead led me to the correct location. I figured it would be fine to keep it for future issues.